src/SmartComponents/Applications/Applications.js

```diff
@@ -1,20 +1,55 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { connect } from 'react-redux';
+import { Skeleton, PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+import FormRender from '@data-driven-forms/react-form-renderer';
+import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import registryDecorator from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+import { register } from '../../store';
+import reducers  from '../../store/reduces.js';
+import { getSchema } from '../../actions';
 
+const localStorageKey = (appName, user) => `@@settings-${appName}-${user}`;
```

Comment:
Can you move this to api file? So we have more reasonable place for all mocked data.